pkg/splunk/enterprise/util.go

```diff
@@ -2160,3 +2168,25 @@ func setProbeLevelOnCRPods(ctx context.Context, cr splcommon.MetaObject, replica
 	}
 	return err
 }
+
+// getApplicablePodNameForK8Probes gets the Pod name relevant for the CR under work
+func getApplicablePodNameForK8Probes(cr splcommon.MetaObject, ordinalIdx int32) string {
```

Comment:
I think you already have an exactly same function `getApplicablePodNameForAppFramework` in afwscheduler.go.

Reply:
nvm ignore it